lib/httpi/adapter/em_http.rb

```diff
@@ -48,6 +50,9 @@ def cert_directory
       # @see HTTPI.request
       def request(method)
         _request { |options| @client.send method, options }
+      rescue EmHttpTimeoutError
+        $!.extend TimeoutError
```

Comment:
It's better to just raise a new exception and if you want to know the source, use `Exception#cause`.